dataCollection/settings.py

```diff
@@ -8,7 +8,7 @@
 #TEMPLATE_DEBUG = DEBUG
 
 ADMINS = (
-    ('Tom Moore', 'tjm3772@rit.edu'),
+    ('Sally PandaFace', 'spf3000@rit.edu'),
```

Comment:
If this info is supposed to change per-site, why is it in version control (note: this is just discussion of previous design decisions)?